connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ErrorHandlingTaskTest.java

```diff
@@ -554,7 +554,7 @@ private ConsumerRecords<byte[], byte[]> records(ConsumerRecord<byte[], byte[]> r
     private abstract static class TestSinkTask extends SinkTask {
     }
 
-    static class FaultyConverter extends JsonConverter {
+    public static class FaultyConverter extends JsonConverter {
```

Comment:
Worth adding a comment on why these are public, so that a future contributor doesn't accidentally revert this change?